stackslib/src/chainstate/coordinator/comm.rs

```diff
@@ -63,8 +63,10 @@ pub struct CoordinatorChannels {
 /// Notification struct for communicating to
 ///  the coordinator. Each bool indicates a notice
 ///  that there are new events of a type to check
+#[derive(Default)]
 struct SignalBools {
     new_stacks_block: bool,
+    new_nakamoto_block: bool,
```

Comment:
Is a new nakamoto block signal necessary? I think it should be able to just use `new_stacks_block`.